website/docs/docs/build/incremental-strategy.md

```diff
@@ -10,32 +10,33 @@ There are various strategies to implement the concept of incremental materializa
 * The reliability of your `unique_key`.
 * The support of certain features in your data platform.
 
-An optional `incremental_strategy` config is provided in some adapters that controls the code that dbt uses
-to build incremental models.
+An optional `incremental_strategy` config is provided in some adapters that controls the code that dbt uses to build incremental models.
 
-### Supported incremental strategies by adapter
```

Comment:
do we need versioning here? (since microbatch is versionless + core 1.9+)

Reply:
Resolving this convo since I double-checked that and saw this line:

> This table represents the availability of each incremental strategy, based on the latest version of dbt Core and each adapter.